packages/types/src/codec/EnumType.ts

```diff
@@ -127,6 +127,30 @@ export default class Enum extends Base<Codec> implements Codec {
     return class extends Enum {
       constructor (value?: any, index?: number) {
         super(Types, value, index);
+
+        Object.keys(this._def).forEach((_key) => {
```

Comment:
`is*` and `as*` getters - basically in the same form as we have for the struct getters

Reply:
And has tests added for these as well.